### ISSUE TITLE
Import cannot handle UTF-characters

### ISSUE THREAD

**Issue by jsuchome** [churchio/onebody#615]
I can export people to csv file, and by looking into that file I can see that the names are correcly shown, including special (language specific) characters (say, Jiří, for example). However, importing the same file back does seem to ignore these characters.

I assume it might be some simple conversion function missing that is present for the export functionality, but forgotten for the import... ?

**Comment by gustavobim** [comment]
I'm had the same issue you described. In order to import special characters I added a new line in the csv.rb file (/app/models/import_parser/strategies/csv.rb) to force the encoding to UTF-8 (added just before the CSV.parse call): `string.force_encoding('UTF-8')`

Also, in my import, some addresses contained the comma value. As a workaround, I'm changing the separator to semicolon. I did that by changing the CSV.parse arguments: `rows = ::CSV.parse(string, headers: true, col_sep: ";")`

This is the parse methods after my changes

      def parse(string)
        string.force_encoding('UTF-8')
        rows = ::CSV.parse(string, headers: true, col_sep: ";")
        {
          headers: rows.headers,
          rows: rows.map(&:to_hash)
        }


@seven1m, I'm thinking to allow each user to choose which separator they would like to use in the import process. For some cases semicolon or tab would work better than comma. Also, the Text Delimiter (") could be another option for some cases. 

Any thoughts on that? Is this something you see as an option in the configurations screen (could have a default value and users would be able to change it in their installs).
